components/api/api-modules-javascript/src/main/resources/META-INF/dirigible/modules/src/io/bytes.ts

```diff
@@ -22,52 +24,53 @@ const JByte = Java.type("java.lang.Byte");
 const JArray = Java.type("java.lang.reflect.Array");
 const BytesFacade = Java.type("org.eclipse.dirigible.components.api.io.BytesFacade");
 
-export function toJavaBytes(bytes) {
-	const internalBytes = JArray.newInstance(JByte.TYPE, bytes.length);
-	for (let i=0; i<bytes.length; i++) {
-		internalBytes[i] = bytes[i];
-	}
-	return internalBytes;
-};
+export class Bytes{
 
-/**
- * Convert the Java byte array to a native JavaScript one. To be used internally by the API layer
- */
-export function toJavaScriptBytes(internalBytes) {
-	const bytes = [];
-	for (let i=0; i<internalBytes.length; i++) {
-		bytes.push(internalBytes[i]);
-	}
-	return bytes;
-};
+   public static toJavaScriptBytes(internalBytes: Array<bytes>): Array<bytes> {
```

Comment:
I'm not entirely sure if we actually need such functions. As far as I remember, GraalJS handles the conversions internally, could you check that?